pkgs/misc/uboot/default.nix

```diff
@@ -496,6 +496,23 @@ in {
     # sf probe; sf update $loadaddr 0 80000
   };
 
+  ubootVisionFive = buildUBoot {
+    version = "2022.04";
+
+    src = fetchFromGitHub {
+      owner = "NickCao";
+      repo = "u-boot-starfive";
+      rev = "ac75aa54020412a83b61dad46c5ea15e7f9f525c";
+      sha256 = "1idh5k1479znp24rrfa0ikgk6iv5h80zscqhi6yv5ah4czia3ip3";
+    };
+
+    # Not sure how to work around this... Seems to be a binutils-related issue?
+    NIX_CFLAGS_COMPILE = "-march=rv64gc_zicsr_zifencei";
+    defconfig = "starfive_jh7100_starlight_smode_defconfig";
```

Comment:
This should be starfive_jh7100_visionfive_smode_defconfig

Reply:
I was looking for that in the u-boot repo but never found it, it compiles but are you sure it's valid?

Reply:
It's here https://github.com/NickCao/u-boot-starfive/blob/master/configs/starfive_jh7100_visionfive_smode_defconfig

Reply:
My u-boot tree is from the starfive fork base onto u-boot v2022.4, upstream u-boot has no support for visionfive yet.

Reply:
Ahh okay, thanks.